mreg_cli/utilities/api.py

```diff
@@ -232,19 +205,23 @@ def _request_wrapper(
 
 
 @overload
-def get(path: str, params: dict[str, Any] | None, ok404: Literal[True]) -> ResponseLike | None: ...
+def get(path: str, params: dict[str, Any] | None, ok404: Literal[True]) -> ResponseLike | None:
+    ...
 
 
 @overload
-def get(path: str, params: dict[str, Any] | None, ok404: Literal[False]) -> ResponseLike: ...
+def get(path: str, params: dict[str, Any] | None, ok404: Literal[False]) -> ResponseLike:
+    ...
 
 
 @overload
-def get(path: str, params: dict[str, Any] | None = ..., *, ok404: bool) -> ResponseLike | None: ...
+def get(path: str, params: dict[str, Any] | None = ..., *, ok404: bool) -> ResponseLike | None:
+    ...
 
 
 @overload
-def get(path: str, params: dict[str, Any] | None = ...) -> ResponseLike: ...
+def get(path: str, params: dict[str, Any] | None = ...) -> ResponseLike:
+    ...
```

Comment:
What caused this? I am using version 2024.20.0 of the Ruff VSCode extension and Ruff 0.4.4, and it does not want to make the changes shown here.

Reply:
No idea, I'll revert it and try to find out.